frontend/src/components/transactions/__tests__/ReceiptRow.test.tsx

```diff
@@ -31,8 +31,8 @@ describe("<ReceiptRow />", () => {
     expect(
       renderer.create(
         <ReceiptRow
-          receipt={(TRANSACTIONS[1].receiptsOutcome || [])[0]}
-          key={(TRANSACTIONS[1].receiptsOutcome || [])[0].id || "ccc"}
+          receipt={TRANSACTIONS[2].receipt!}
```

Comment:
I realized that `TRANSACTIONS[2]` has no failing receipts, which is what this test title states. I suggest we split `TRANSACTIONS` into independent constants (e.g. `TRANSACTION_WITH_FAILING_RECEIPTS`, `TRANSACTION_WITH_HUGE_AMOUNTS`, etc) since otherwise, it is easy to introduce a completely inadequate test.

The test was clearly broken before, but I would like to ask you to fix this situation while we are at it.